### DISCUSSION TITLE
default_gui_startup_args

### DISCUSSION THREAD

**Discussion by klapvogn** [wezterm/wezterm#6030] [General]
Hi,

I use Windows

Is it possible to do something like this. I mainly use the Wezterm to connect to a remote server:
`config.default_gui_startup_args = { 'ssh', '-oIdentityFile=path\to\id_ed25519', 'USER@IP:50512' }`

**Comment by nekowinston** [comment]
I'd suggest using [default_domain](https://wezfurlong.org/wezterm/config/lua/config/default_domain.html) and [SSH Domains](https://wezfurlong.org/wezterm/config/lua/SshDomain.html) instead, e.g.:

```lua
config.ssh_domains = {
	{
		name = "my-server",
		remote_address = "192.168.1.1:2222",
		username = "user",
		multiplexing = "None",
		ssh_option = {
			identityfile = "/path/to/id_rsa.pub",
		},
	},
}
config.default_domain = "my-server"
```

**Reply by klapvogn** [reply]
thanks :)
